software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/couchdb/CouchDBNodeSshDriver.java

```diff
@@ -129,7 +199,7 @@ public void customize() {
     public void launch() {
         log.info("Launching  {}", entity);
         newScript(MutableMap.of(USE_PID_FILE, false), LAUNCHING)
-                .body.append(sudo(String.format("nohup couchdb -p %s -a %s -o couchdb-console.log -e couchdb-error.log -b &", getPidFile(), Os.mergePathsUnix(getRunDir(), getCouchDBConfigFileName()))))
+                .body.append(String.format("nohup ./bin/couchdb -p %s -a %s -o couchdb-console.log -e couchdb-error.log -b &", getPidFile(), Os.mergePathsUnix(getRunDir(), getCouchDBConfigFileName())))
```

Comment:
The bin folder string is used in some different parts. Would it make sense to wrap `/bin/couchdb` folder with a method? 
Moreover, I looks that couchdb files are installed over different folders:
`/usr/bin/couchdb` contains the launch ssh file, and `/etc/couchdb` contains some configuration files  and `/usr/share/couchdb` contains the source to execute the server. Then, would it make sense try to store these paths in any variable or sensor? It looks that these values are concrete values of this driver, but  may it be useful for future improvement?